peopleops: Require memos for Brex (#111)
flowforge/handbook: peopleops/index.md

default, virtual Mastercards, although there is an option to get a physical
card sent out.

> All expenses on Brex require a memo to be submitted what the expense if for, and
> expenses made equal or more than $25 require a receipt to be uploaded. Attaching
> a receipt to an expense can be done through their web UI, forwarding receipts
> through email, or through Slack. For full explanation, please [read their docs][brex-receipts].

Reviewer: ```suggestion
All expenses on Brex require a memo to be submitted with details on what the expense is for.
All expenses for $25 or more require a receipt to be uploaded. Attaching
a receipt to an expense can be done through their web UI, forwarding receipts
through email, or through Slack. For a full explanation, please [read their docs][brex-receipts].
```

Reviewer: Minor change to improve clarity.
